metrics/web/lib/common/presentation/navigation/route_configuration/route_configuration_location_converter.dart

```diff
@@ -0,0 +1,28 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:metrics/common/presentation/navigation/route_configuration/route_configuration.dart';
+
+/// A class that provides a method for converting the given [RouteConfiguration]
+/// to a location [String].
```

Comment:
🏷️
```suggestion
/// A class that provides a method for converting the [RouteConfiguration]
/// to a location [String].
```
What do you think?